lib/dry/web/roda/templates/Rakefile.tt

```diff
@@ -15,11 +15,30 @@ def db
   <%= config[:camel_cased_app_name] %>::Container["persistence.db"]
 end
 
+def settings
+  <%= config[:camel_cased_app_name] %>::Container["settings"]
+end
+
+def execute_database_create_command(settings)
+  database_url = settings.database_url
+  _, _, _, db_name = database_url.split('/')
+  if user_and_net.include?('@')
```

Comment:
Yes, you are right I definitely left out that, part. I would change it anyway to use `URI`